AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.java

```diff
@@ -2090,6 +2090,17 @@ private void openStudyOptions(boolean withDeckOptions) {
         }
     }
 
+    private void shouldOpenReviewerOnSelection(boolean dontSkipStudyOptions) {
```

Comment:
```suggestion
    private void openReviewerOrStudyOptions(boolean dontSkipStudyOptions) {
```

Just a name suggestion so it is aligned with what is actually happening in this commit

Reply:
Fine by me